Packs/Base/ReleaseNotes/1_0_6.md

```diff
@@ -2,3 +2,4 @@
 #### Scripts
 - __CommonServerPython__
   - Added support for the CVE indicator class.
+  - Formatting and lint improvements.
```

Comment:
This should be in a new release notes file 1_0_7.md. 1.0.6 is already released.

Reply:
it's the result of multiple people working on the same pack. Maybe we should extract the CommonServer files into a separate pack.